lib/msal-browser/src/interaction_client/StandardInteractionClient.ts

```diff
@@ -171,6 +172,16 @@ export abstract class StandardInteractionClient extends BaseInteractionClient {
         };
     }
 
+    private static createDefaultLoggerOptions(): LoggerOptions {
```

Comment:
This should be done in `Configuration.ts` instead

Reply:
Addressed